### PR TITLE
Complete Project Upload (Library + 5 Demos)

### PR DESCRIPTION
**Pull request by holy-script**
As instructed, library folder is separate and installable after repository is clones locally.

### CODE REVIEW

File: entry/build/intermediates/javac/debug/classFiles [hunk 1]
```diff
@@ -0,0 +1,4 @@
+C:\Users\Palash\DevEcoStudioProjects\JSVerbalExpressionsHMOS\entry\build\intermediates\javac\debug\classes\com\example\jsverbalexpressionshmos\BuildConfig.class
```
Comment (archanaatr) [inline entry/build/intermediates/javac/debug/classFiles]: Please remove the unwanted folders like build

File: entry/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3212 @@
...
+        "node": ">=8"
+      }
+    },
+    "node_modules/lodash": {
+      "version": "4.17.21",
+      "resolved": "https://repo.huaweicloud.com/repository/npm/lodash/-/lodash-4.17.21.tgz",
+      "integrity": "sha512-v2kDEe57lecTulaDIuNTPy3Ry4gLGJ6Z1O3vE1krgXZNrsQ+LFTGHVxVjcXPs17LhbZVGedAJv8XZ1tvj5FvSg==",
+      "license": "MIT"
+    },
+    "node_modules/lodash-id": {
```
Comment (archanaatr) [inline entry/package-lock.json]: many node_modules added in this file, please recheck

File: entry/src/main/js/default/i18n/en-US.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "strings": {
+    "hello": "Hello",
```
Comment (archanaatr) [inline entry/src/main/js/default/i18n/en-US.json]: remove the unwanted strings

**Review by archanaatr — COMMENTED** [review]
Please check the comments

File: entry/src/main/js/default/pages/index/index.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
+#page {
```
Comment (archanaatr) [inline entry/src/main/js/default/pages/index/index.css]: change the style names to .page. .card etc.,   to start with dot and test once in emulator.. if any unused ones can be removed.

**Review by archanaatr — COMMENTED** [review]
Please check the comments.

**Comment by holy-script** [comment]
Updated files according to reviews

**Review by archanaatr — APPROVED** [review]
Ok

**Pull request merged**